RELEASES.md

```diff
@@ -1,3 +1,61 @@
+Version 1.21.0 (2017-10-12)
+==========================
+
+Language
+--------
+- [Relaxed path syntax. You can now add type parameters to values][43540]
+  Example: `struct Foo<T>(T); let bar = Foo::<u8>(0);`.
```

Comment:
`Foo::<u8>(0)` was always possible, what https://github.com/rust-lang/rust/pull/43540 does is permitting (sometimes unnecessary) `::` before `<` in all paths, including paths passed to macros.
Previously:
```rust
my_macro!(Vec<i32>::new); // OK
my_macro!(Vec::<i32>::new); // ERROR
```
Now:
```rust
my_macro!(Vec<i32>::new); // OK
my_macro!(Vec::<i32>::new); // OK too
```